src/sdk/object_sdk.js

```diff
@@ -801,8 +801,22 @@ class ObjectSDK {
     ////////////
 
     async list_buckets() {
-        const bs = this._get_bucketspace();
-        return bs.list_buckets(this);
+        const start_time = Date.now();
+        let error = 0;
```

Comment:
As discussed in the last review,
these lines should be a wrapper of each function that requires this metrics logic, instead of having this logic 1 time, it needs to be maintained in ~10 different functions.

Reply:
and every function has 16 rows instead of 2

Reply:
This should be a handler, but as for now, We will keep it this way and make a PR for creating a handler later.

Reply:
@romayalon I talked to @nimrod-becker 
We will do that for 4.13, and after the point that we don't plan to add more ops for 4.12.z